examples/image_reconstruction.ipynb

```diff
@@ -0,0 +1,705 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "4rGqr2VSyqrK",
+    "outputId": "2537ca9d-ee9f-4aeb-e91d-e148e5827e3f"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: poutyne in /usr/local/lib/python3.6/dist-packages (1.2)\n",
+      "Requirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from poutyne) (1.19.5)\n",
+      "Requirement already satisfied: torch in /usr/local/lib/python3.6/dist-packages (from poutyne) (1.7.0+cu101)\n",
+      "Requirement already satisfied: dataclasses in /usr/local/lib/python3.6/dist-packages (from torch->poutyne) (0.8)\n",
+      "Requirement already satisfied: future in /usr/local/lib/python3.6/dist-packages (from torch->poutyne) (0.16.0)\n",
+      "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.6/dist-packages (from torch->poutyne) (3.7.4.3)\n"
+     ]
+    }
+   ],
+   "source": [
+    "pip install poutyne"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Uwt_Nl1Qw_Pz"
+   },
+   "outputs": [],
+   "source": [
+    "#importing the needed libraries\n",
+    "\n",
+    "%matplotlib inline\n",
+    "%reload_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "import numpy as np\n",
+    "import math\n",
+    "import os\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "\n",
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "import torch.optim as optim\n",
+    "from torch.utils.data import DataLoader,Subset, Dataset\n",
+    "\n",
+    "\n",
+    "import torchvision\n",
+    "import torchvision.datasets as datasets\n",
+    "import torchvision.transforms as tfms\n",
+    "from torchvision.utils import make_grid\n",
+    "\n",
+    "\n",
+    "import poutyne\n",
+    "from poutyne import set_seeds, Model, Experiment, ModelCheckpoint,CSVLogger\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "j7N7G1IWw_P0",
+    "outputId": "3f03c676-b228-44cb-95a3-594b7e3abc9b"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The running processor is... cuda\n"
+     ]
+    }
+   ],
+   "source": [
+    "# hyper parameters\n",
```

Comment:
```suggestion
    "# Hyperparameters\n",
```

Reply:
I think this should be a title or under something like this https://poutyne.org/examples/introduction.html#training-constants